docs/serving/samples/knative-routing-go/README.md

```diff
@@ -268,7 +268,7 @@ The Gateway proxy checks the updated host, and forwards it to `Search` or
 
 ## Using internal services and `"httpProtocol": "Redirected"`
 
-Using the above approach, services will be available using two entrypoints into the cluster:
+Using the mentioned approach, services will be available using two entrypoints into the cluster:
```

Comment:
```suggestion
Using the approach explained in the previous section, services will be available using two entrypoints into the cluster:
```

Reply:
It would be clearer to say when this approach was mentioned. The suggestion above assumes that it is referring to the previous section but I wasn't sure.